source/renderer/app/stores/HardwareWalletsStore.ts

```diff
@@ -682,11 +726,11 @@ export default class HardwareWalletsStore extends Store {
 
           if (this.isExportKeyAborted) {
             if (isTrezor) {
-              await this._getExtendedPublicKey(
-                recognizedPairedHardwareWallet.path,
-                activeWalletId,
-                this.unfinishedWalletAddressVerification
-              );
+              await this._identifyAndHandleSoftwareWallet({
```

Comment:
@szymonmaslowski This should remain `_getExtendedPublicKey` because that is the main procedure and we are identifying Hardware-Software just to check if we already have an extended public key that matches some software wallet

Reply:
Ouu, I see now that you have dedicated fn for `_getExtendedPublicKey` and this is just for matching. 
Ignore the comment! Resolved!